### PR TITLE
mame: 0.221 -> 0.222

### PR DESCRIPTION
**Pull request by ghost**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/misc/emulators/mame/default.nix [hunk 1]
```diff
@@ -26,7 +26,7 @@ in mkDerivation {
     owner = "mamedev";
     repo = "mame";
     rev = "mame${majorVersion}${minorVersion}";
-    sha256 = "1pqwa5qcrk0wgbyns6fkv7j3pn7byf216cixc3phhhgvfidgr8ww";
+    sha256 = "1jwc50hmlrkdj204gqcvf7lvgb1r6rhj8hhckykylb1mli04c7gy";
```
Comment (danieldk) [inline pkgs/misc/emulators/mame/default.nix:29]: Both ofborg and my local machine get a different hash, namely:

```suggestion
    sha256 = "1ij08h7cflr76qzyhhj21948275lqkpzi9r4pbc7h2avrlpsijx4";
```

**Review by danieldk — CHANGES_REQUESTED** [review]

File: pkgs/misc/emulators/mame/default.nix (hunk 1 above)
Comment (gnidorah) [reply pkgs/misc/emulators/mame/default.nix]: @danieldk Applied, thanks!

**Review by danieldk — APPROVED** [review]
LGTM

Result of `nixpkgs-review pr 91618` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
<br>- mame
</details>

`mame64` binary runs as expected. Thanks!

**Comment by danieldk** [comment]
Jikes, missed that the hash change was now a separate commit. The commits should've been squashed into once. Better luck next time!

**Comment by gnidorah** [comment]
@danieldk Thank you!

**Pull request merged**